audioutils/.gitignore

```diff
@@ -0,0 +1 @@
+/Kconfig
```

Comment:
why gitignore is needed?

Reply:
It is because of nuttx build system.
The build system makes Kconfig automatically with full path.

gpsutils takes the same way.

Reply:
Ok, so this is for autogenerated Kconfig. That's fine.